ygot/types.go

```diff
@@ -37,6 +37,16 @@ type ValidatedGoStruct interface {
 	Validate() error
 }
 
+// KeyHelperGoStruct is an interface which can be implemented by Go structs
+// that are generated to represent a YANG container or list member that has
+// the corresponding function to retrieve the list keys as a map.
+type KeyHelperGoStruct interface {
+	GoStruct // Must be a valid GoStruct
```

Comment:
Just to avoid linter problems down the road, suggest putting this as // GoStruct blah above.

Reply:
Done.